tests/unit/python/foglamp/common/test_configuration_manager.py

```diff
@@ -132,7 +132,11 @@ def test_unregister_interest(self, reset_singleton):
         with patch.object(_logger, "error") as log_error:
             with pytest.raises(Exception) as excinfo:
                 await c_mgr._run_callbacks('name')
-            assert excinfo.type is ImportError
+            import sys
+            if sys.version_info[1] >= 6:
+                assert excinfo.type is ModuleNotFoundError
+            else:
+                assert excinfo.type is ImportError
```

Comment:
@ashwinscale  @MarkRiddoch  I hope you have tested south server plugin import error handling too!